src/plugins/home/public/application/components/homepage/sections/learn_basics.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { FC } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCard,
+  EuiIcon,
+  EuiListGroup,
+  EuiListGroupItemProps,
+  IconType,
+} from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+import { Section } from '../../../../services/section_type/section_type';
+import { renderFn } from './utils';
+import { getServices } from '../../../opensearch_dashboards_services';
+
+const Card: FC<{
+  iconType: IconType;
+  title: string;
+  listItems: EuiListGroupItemProps[];
+}> = ({ iconType, title, listItems }) => (
+  <EuiFlexItem grow={1}>
+    <EuiCard
+      icon={<EuiIcon type={iconType} size="xl" />}
+      title={
+        <>
+          {title} <EuiIcon type="popout" />
+        </>
+      }
+      layout="horizontal"
+    >
+      <EuiListGroup className="home-basics-listGroup" listItems={listItems} />
+    </EuiCard>
+  </EuiFlexItem>
+);
+
+const render = renderFn(() => {
+  const services = getServices();
+  const logos = services.chrome.logos;
+
+  const gettingStartedLinks: EuiListGroupItemProps[] = [
+    {
+      label: i18n.translate('home.sections.learnBasics.gettingStarted.quickstart', {
+        defaultMessage: 'OpenSearch Dashboards quickstart',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/quickstart/',
+      target: '_blank',
+    },
+    {
+      label: i18n.translate('home.sections.learnBasics.gettingStarted.dataVis', {
+        defaultMessage: 'Building data visualizations',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/visualize/viz-index/',
+      target: '_blank',
+    },
+    {
+      label: i18n.translate('home.sections.learnBasics.gettingStarted.dashboards', {
+        defaultMessage: 'Creating dashboards',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/dashboard/index/',
+      target: '_blank',
+    },
+  ];
+
+  const dataDiscoveryLinks: EuiListGroupItemProps[] = [
+    {
+      label: i18n.translate('home.sections.learnBasics.dataDiscovery.getFamiliar', {
+        defaultMessage: 'Get familiar with Discover',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/discover/index-discover/',
+      target: '_blank',
+    },
+    {
+      label: i18n.translate('home.sections.learnBasics.dataDiscovery.runQueries', {
+        defaultMessage: 'Run queries in the Dev Tools Console',
+      }),
+      href: 'https://opensearch.org/docs/latest/dashboards/dev-tools/run-queries/',
+      target: '_blank',
+    },
+    {
+      label: i18n.translate('home.sections.learnBasics.dataDiscovery.indexes', {
+        defaultMessage: 'Working with indexes',
```

Comment:
i always thought it was weird how the API is `indices` but our documentation references `indexes`. perhaps something we should ping doc team for

Reply:
OSD and the docs use both forms. I pinged the good people at @opensearch-project/docs to ask about this.

Reply:
According to https://github.com/opensearch-project/documentation-website/blob/main/TERMS.md, we should use `indexes` for anything new we introduce; existing ones should be left as they are.

Reply:
Just to clarify, we should use "indexes" in text but for variables, links, settings, etc., we can keep `indices`. So the APIs use `indices` but the descriptions say "indexes" :)